src/test/java/io/cdap/plugin/hubspot/common/BaseETLTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.hubspot.common;
+
+import io.cdap.cdap.etl.mock.test.HydratorTestBase;
+
+import javax.annotation.Nullable;
+
+public abstract class BaseETLTest extends HydratorTestBase {
+    @Nullable
```

Comment:
nit: Indentation for this file seems incorrect.

Reply:
Fixed